### PR TITLE
Add Kubernetes, cadvisor and Bosun to Monitoring & Administration

### PR DESCRIPTION
**Pull request by PierreZ**

### CODE REVIEW

**Comment by nodiscc** [comment]
Thanks @PierreZ; can you change all occurences of `SourceCode` to `Source code` (whitespace, casing)?

**Comment by PierreZ** [comment]
Yes, I'll do that on tuesday

File: README.md [hunk 1]
```diff
@@ -376,9 +376,12 @@ Some [Groupware](#groupware) solutions also feature file sharing synchronization
...
   * [adminer](http://www.adminer.org/) -  Database management in a single PHP file ([Source code](https://github.com/vrana/adminer))
-  * [Icinga / Icinga 2](https://www.icinga.org/) - Nagios fork that has since lapped nagios several times. Comes with the possibility of clustered monitoring - ([SourceCode](https://github.com/Icinga)) - `GPLv2`
+  * [Bosun](http://bosun.org/) - Monitoring and alerting system by Stack Exchange ([Source Code](https://github.com/bosun-monitor/bosun), [Documentation](http://bosun.org/quickstart.html)) `MIT` `Go`
+  * [cadvisor](https://github.com/google/cadvisor) - Analyzes resource usage and performance characteristics of running containers ([Source Code](https://github.com/google/cadvisor)) `Apache` `Go`
+  * [Icinga / Icinga 2](https://www.icinga.org/) - Nagios fork that has since lapped nagios several times. Comes with the possibility of clustered monitoring - ([Source Code](https://github.com/Icinga)) - `GPLv2`
+  * [Kubernetes](http://kubernetes.io/) - Orchestration system for Docker containers - ([Source Code](https://github.com/kubernetes/kubernetes), [Documentation](http://kubernetes.io/v1.0/)) `Apache` `Go`
```
Comment (nodiscc) [inline README.md]: I don't think kubernetes belongs to this list, it's an administration tool (not a network service/web application). I think we should remove it. @PierreZ you might want to submit it to https://github.com/n1trux/awesome-sysadmin

**Comment by nodiscc** [comment]
As discussed in https://github.com/Kickball/awesome-selfhosted/issues/214 and https://github.com/Kickball/awesome-selfhosted/issues/127, we should really move those entries to https://github.com/n1trux/awesome-sysadmin. 

@PierreZ we are planning to move the admin/monitoring section to awesome-sysadmin. Can you submit your entries there? Icinga is already in https://github.com/n1trux/awesome-sysadmin#monitoring. For the rest I think the maintainer appreciates a quick explanation/overview of the software in the Pull Request.

**Comment by nodiscc** [comment]
Moved to https://github.com/n1trux/awesome-sysadmin/issues/105

**Pull request closed without merging**